01_prepare_host.sh

```diff
@@ -57,6 +57,7 @@ elif [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
         curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/"${KIND_VERSION}"/kind-"$(uname)"-amd64
         chmod +x ./kind
         sudo mv kind /usr/local/bin/.
+        sudo "${CONTAINER_RUNTIME}" pull kindest/node:"${KUBERNETES_VERSION}"
```

Comment:
This is the target cluster kubernetes version. I don't think we want to tie them together. depending on the kind version installed, it will probably use another image and hence download it again. you can either download the correct image based on the kind version, or give the --image parameter to take this specific image into use

Reply:
I was thinking to create a new var `KIND_IMAGE_VERSION` but then I saw that we are already using `KUBERNETES_VERSION` var when running [kind create](https://github.com/metal3-io/metal3-dev-env/blob/b156fcafcf47b982c4cc5ad8c08d55d60d592b6e/03_launch_mgmt_cluster.sh#L191). Also, I have tested locally, and even if the version of the Kind image is not the one that is downloaded by default depending on the Kind version you still can run that image.
I'm fine with both option. I can create a new var or rely on ` KUBERNETES_VERSION`

Reply:
oh, I didn't see that we are already forcing the image in the create, sorry. Then it is fine as is